identity-service/src/notifications/index.js

```diff
@@ -210,7 +210,7 @@ class NotificationProcessor {
    * 5. Process milestones
    * @param {Integer} minBlock min start block to start querying discprov for new notifications
    */
-  async indexAll (audiusLibs, minBlock, oldMaxBlockNumber) {
+  async indexAll (audiusLibs, expressApp, minBlock, oldMaxBlockNumber) {
```

Comment:
I think it would be a bit cleaner to pass the optimizelyClient down rather than the entire expressApp - would separate concerns, like how we try to avoid passing requests around in CN.